proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/Const.java

```diff
@@ -160,22 +158,22 @@
         mandatoryStreams.add(ALTO_GRP_ID);
         mandatoryStreams.add(TXT_GRP_ID);
 
-        streamMapping = new HashMap<String, List<String>>();
-        streamMapping.put(MC_GRP_ID, new ArrayList<String>());
+        streamMapping = new HashMap<>();
+        streamMapping.put(MC_GRP_ID, new ArrayList<>());
         streamMapping.get(MC_GRP_ID).add(BinaryEditor.NDK_ARCHIVAL_ID);
         // streamMapping.get("MC_IMGGRP").add("RAW");
 
-        streamMapping.put(UC_GRP_ID, new ArrayList<String>());
+        streamMapping.put(UC_GRP_ID, new ArrayList<>());
         streamMapping.get(UC_GRP_ID).add(BinaryEditor.NDK_USER_ID);
         // streamMapping.get("UC_IMGGRP").add("FULL");
 
-        streamMapping.put(ALTO_GRP_ID, new ArrayList<String>());
+        streamMapping.put(ALTO_GRP_ID, new ArrayList<>());
         streamMapping.get(ALTO_GRP_ID).add(AltoDatastream.ALTO_ID);
 
-        streamMapping.put(TXT_GRP_ID, new ArrayList<String>());
+        streamMapping.put(TXT_GRP_ID, new ArrayList<>());
         streamMapping.get(TXT_GRP_ID).add(StringEditor.OCR_ID);
 
-        streamMapping.put("TECHMDGRP", new ArrayList<String>());
+        streamMapping.put("TECHMDGRP", new ArrayList<>());
```

Comment:
Je `TECHMDGRP` kvůli něčemu jako hodnota?

Reply:
To není můj kód, ale mezi těmi ostatními je to do očí bijící, že...i když to idea jako magic constant neoznačila. Změním to.

Celá ta třída s konstantama mi přijde divná - nemám rád tyto classy s konstama, myslím, že je lepší, když se konstanta nadefinuje ve třídě která ji potřebuje.
Další problém je, že ty arraylisty jsou mutable, ale teď to refaktorovat asi nebudu.